pkg/tinycbor/Makefile

```diff
@@ -1,6 +1,7 @@
 PKG_NAME=tinycbor
 PKG_URL=https://github.com/intel/tinycbor
-PKG_VERSION=v0.5.1
+# Tinycbor v0.5.2
+PKG_VERSION=d94ca09aa91f5b3c581527aa8bca179a82b79874
```

Comment:
Wouldn't it be preferrable to stick to a release tag instead of the commit hash?

Reply:
IIRC the consensus is to use a commit hash here. The argument is that a version tag can be changed to point to a different commit while that is (practically) impossible with a commit hash.

Reply:
If this was the consensus, fine with me, especially since you add the version as a comment. However, I would argue to drop immediately the support for a repository that modifies a tag. ;-) (Refer to the "Re-Tagging" section in https://git-scm.com/docs/git-tag.)

Reply:
As we are re-downloading the repository everytime, we would not detect it (not sure about what git-cache does).

And in case of a compromised upstream, I think we would build it in CI without noticing the change.

Reply:
@miri64 @cladmi Do you want me to open an issue for this? Describe problem, get some opinions and call for a vote after some time?

In the mean time I'd like to keep the commit hash here to have it consistent with the other packages.

Reply:
Sounds like a reasonable approach to me. Please go ahead!

Reply:
See #9966